astropy_helpers/extern/automodapi/automodapi.py

```diff
@@ -381,8 +383,13 @@ def process_automodapi(app, docname, source):
 
 
 def setup(app):
-    # need automodsumm for automodapi
-    app.setup_extension('astropy_helpers.sphinx.ext.automodsumm')
+
+    app.setup_extension('sphinx.ext.autosummary')
+
+    # Note: we use __name__ here instead of just writing the module name in
+    #       case this extension is bundled into another package
+    from . import automodsumm
```

Comment:
This is a bit confusing: doesn't this mean it's *always* going to be the bundled one, regardless of whether another copy is installed?  Or is that the *intended* behavior, and a later change would then make an "external" copy possible?